hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/validators/TokenUpdateValidator.java

```diff
@@ -62,6 +62,10 @@ public ValidationResult validateSemantics(
         if (op.hasMemo()) {
             context.attributeValidator().validateMemo(op.memo());
         }
+        // validate metadata
+        if (op.hasMetadata()) {
+            validator.validateTokenMetadata(op.metadata(), tokensConfig);
```

Comment:
We could generalize `context.attributeValidator().validateMemo(` and re-use it here